addons/rose/app/styles/rose/components/_index.scss

```diff
@@ -4,18 +4,18 @@
  */
 
 @import 'button';
+@import 'card';
```

Comment:
None blocking comment: What is the reason behind the re-ordering?

Reply:
I think alphabetizing lists improves readability

Reply:
Great advice! ✨